frontend/src/components/pages/ForgotPassword.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from "react";
+import {
+  Box,
+  Heading,
+  Input,
+  Text,
+  Button,
+  FormControl,
+  FormErrorMessage,
+} from "@chakra-ui/react";
+import background from "./login_background.png";
+import backgroundMobile from "./login_background_phone.png";
+
+type SentEmail = {
+  email: string;
+  timestamp: number;
+};
+
+const ForgotPassword = (): React.ReactElement => {
+  const [validUser, setValidUser] = useState(true);
+  const [sentEmail, setSentEmail] = useState(false);
+  const [sentEmailToUser, setSentEmailToUser] = useState(false);
+  const [userEmailId, setUserEmaild] = useState("");
+
+  const handleUserAuth = (userEmail: string) => {
+    const emailPattern = /^[^\s@]+@humanesociety\.org$/;
+    const sentEmails: SentEmail[] = JSON.parse(localStorage.getItem("sentEmails") || "[]");
+    if (!emailPattern.test(userEmail)) {
+      setValidUser(false);
+    } else if (sentEmails.some(item => item.email === userEmail)) {
+      setValidUser(true);
+      setSentEmail(false);
+      setSentEmailToUser(true);
+    } else {
+      // make API call to check if user exists
+      setValidUser(true);
+      setSentEmail(true);
+      // send email logic
+      const newEmail: SentEmail = {
+        email: userEmail,
+        timestamp: new Date().getTime()
+      }
+      sentEmails.push(newEmail);
+      localStorage.setItem("sentEmails", JSON.stringify(sentEmails));
+      setTimeout(() => {
+        const updatedSentEmails: SentEmail[] = JSON.parse(localStorage.getItem("sentEmails") || "[]");
+        const filteredEmails = updatedSentEmails.filter(
+          (item: SentEmail) => item.email !== userEmail
+        );
+        localStorage.setItem("sentEmails", JSON.stringify(filteredEmails));
+        setSentEmailToUser(false)
+      }, 60000);
+    }
+  };
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setValidUser(true)
+    setSentEmail(false);
+    setSentEmailToUser(false);
+    setUserEmaild(e.target.value);
+  };
+
+  return (
+    <Box
+      height="100vh"
+      backgroundImage={[
+        `url(${backgroundMobile})`,
```

Comment:
Similar issue here -- if you're on mobile but in landscape, you're going to be using the vertical pawprint background even though it's in landscape